2024/script.txt

```diff
@@ -0,0 +1,79 @@
+T: Good morning, I’m Troy
+
+C: I’m Christian
+
+T & C: and this is UIF News
+
+C: Well Troy, it’s April 1st and you know what that means?
+
+T: It’s Q2
+
+C: Yes that’s right, it’s Q2 2024, even though I’m pretty sure yesterday was January 15th 2023
+
+T: Time flies when you’re having Helix
+
+C: It sure does, but speaking of 2023 that brings us to our top story.
+
+[cut to solo medium shot of christian with dramatic pivot to secondary camera]
+C: In what may feel like a scene straight out of classic 1993 film Groundhog Day, the UIF roadmap for 2024 is looking 
+alarmingly similar to the initial proposal for the 2023 roadmap.
+C: There have been a “growth” of concerns around the similarity, leaving some “division” among UIF members. 
+C: Some are questioning if this was a foreseeable and preventable outcome, while Friedrich Nietzsche maintains 
+that time is a flat circle and we are doomed to endlessly repeat our actions
+T: [Off-camera] Nietzsche? Who let him on the team?
+C: Shut up Troy, this is my piece // can workshop interjection
```

Comment:
the goal is to later make a callback to "this is my piece" with a play on words. but i don't know if "shut up troy" or something like "hey now, this is my piece" is better. we can play around with delivery

Reply:
decided not to do this as we took out the later callback